api/src/main/java/org/openmrs/module/reporting/cohort/query/db/hibernate/HibernateCohortQueryDAO.java

```diff
@@ -787,48 +720,49 @@ public Cohort getPatientsInStates(List<ProgramWorkflowState> states, Date onOrAf
 			query.setTimestamp("onOrAfter", onOrAfter);
 		if (onOrBefore != null)
 			query.setTimestamp("onOrBefore", DateUtil.getEndOfDayIfTimeExcluded(onOrBefore));
-		return new Cohort(query.list()); 
-    }
-
+		return new Cohort(query.list());
+	}
+	
 	/**
-	 * @see org.openmrs.module.reporting.cohort.query.db.CohortQueryDAO#getPatientsHavingRangedObs(org.openmrs.api.PatientSetService.TimeModifier, org.openmrs.Concept, org.openmrs.Concept, java.util.Date, java.util.Date, java.util.List, java.util.List, org.openmrs.module.reporting.common.RangeComparator, java.lang.Object, org.openmrs.module.reporting.common.RangeComparator, java.lang.Object)
+	 * @see org.openmrs.module.reporting.cohort.query.db.CohortQueryDAO#getPatientsHavingRangedObs(org.openmrs.api.PatientSetService.TimeModifier,
+	 *      org.openmrs.Concept, org.openmrs.Concept, java.util.Date, java.util.Date,
+	 *      java.util.List, java.util.List, org.openmrs.module.reporting.common.RangeComparator,
+	 *      java.lang.Object, org.openmrs.module.reporting.common.RangeComparator, java.lang.Object)
 	 */
 	public Cohort getPatientsHavingRangedObs(TimeModifier timeModifier, Concept question, Concept groupingConcept,
-                                              Date onOrAfter, Date onOrBefore,
-                                              List<Location> locationList, List<EncounterType> encounterTypeList,
-                                              RangeComparator operator1, Object value1,
-                                              RangeComparator operator2, Object value2) {
+	        Date onOrAfter, Date onOrBefore, List<Location> locationList, List<EncounterType> encounterTypeList,
+	        RangeComparator operator1, Object value1, RangeComparator operator2, Object value2) {
 		
-		return getPatientsHavingObs(timeModifier, question, groupingConcept, onOrAfter, onOrBefore, locationList, encounterTypeList, operator1, value1, operator2, value2, null, null);
+		return getPatientsHavingObs(timeModifier, question, groupingConcept, onOrAfter, onOrBefore, locationList,
+		    encounterTypeList, operator1, value1, operator2, value2, null, null);
 	}
 	
 	/**
-	 * Encapsulates the common logic between getPatientsHavingRangedObs and getPatientsHavingDiscreteObs
-	 * 
-	 * The arguments passed in fall into two types:
+	 * Encapsulates the common logic between getPatientsHavingRangedObs and
+	 * getPatientsHavingDiscreteObs The arguments passed in fall into two types:
 	 * <ol>
-	 *     <li>arguments that limit which obs we will look at</li>
-	 *     <ul>
-	 *         <li>timeModifier</li>
```

Comment:
@mseaton another more meaningful thought is getting back these lines indented as they looked before which is more meaningful rather than every thing other little code format that has been done since most of such are very better